soc/xtensa/intel_adsp/tools/cavstool.py

```diff
@@ -648,7 +657,7 @@ async def _main(server):
     log.info(f"Detected cAVS {'1.5' if cavs15 else '1.8+'} hardware")
 
     if args.log_only:
-        wait_fw_entered()
+        wait_fw_entered(timeout_s=None)
```

Comment:
Do we really need to wait "forever"?  Surely something eventually has to time out and give up, right?  Or is the expectation that that happens externally in a CI rig or whatever?

Reply:
Yes the idea is really to `--log-only` forever. Either in interactive use and then Ctrl-C is what's expected, or in a "CI rig" and then there will be plenty of other timeouts in various other layers, really no need to go and try to guess what a good default would be there. I mean no way to predict all the test scenarios this small cog will be a part of.

For instance https://github.com/thesofproject/sof-test is already used to leave sof-logger running forever in the background and kill it when it's done (and before starting a new test).